acceptance-tests/src/main/java/io/blueocean/ath/pages/blue/DashboardPage.java

```diff
@@ -0,0 +1,40 @@
+package io.blueocean.ath.pages.blue;
+
+import io.blueocean.ath.BaseUrl;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.PageFactory;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+
+@Singleton
+public class DashboardPage {
+    @Inject
+    WebDriver driver;
+
+    @Inject @BaseUrl
+    String base;
+
+    @Inject
+    public DashboardPage(WebDriver driver) {
+        PageFactory.initElements(driver, this);
+    }
+
+    public void open() {
+        driver.get(base + "/blue/");
+    }
+
+    public boolean isFavorite(String job) {
+        WebElement tr = driver.findElement(By.xpath("//tr[@data-name=\"" + job + "\"]"));
+        WebElement favorite = tr.findElement(By.cssSelector(".Checkbox.Favorite > label > input"));
+        return favorite.isSelected();
```

Comment:
I was more going for a POC, while i was working out Guice, Webdriver etc. Feel free to add to this PR if you want, or wait until its in master. I;m working on a github creation test currently